src/app/offers/single-offer-item/single-offer-item.component.scss

```diff
@@ -8,13 +8,20 @@
   background: $background-color;
   padding-bottom: 10px;
   transition: all 0.3s cubic-bezier(.25, .8, .25, 1);
-  cursor: pointer;
 }
 
 .card-right .app-basic-title {
   display: none;
 }
 
+.app-basic-title, .left__progress, .card-right {
+  padding: 0 20px;
+}
```

Comment:
Why this padding? We already have padding-left: 15px and padding-right: 15px on these elements.

Reply:
Looks like this isn't my code (picture in attachment) Maybe this was added after merging with the master branch. Can you please check it before I delete it?

![Screenshot_1](https://user-images.githubusercontent.com/11729076/102538564-b8911300-40ac-11eb-9bb2-d847e5f3b636.png)

Reply:
I don't have this code in the latest master branch. Guess something went wrong during
merging. It should be ok to remove it. Did you add cursor: pointer on title or was this there before as well?
I don't have that CSS as well.